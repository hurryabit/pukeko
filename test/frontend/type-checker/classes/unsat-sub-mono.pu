class C a where
  c : a -> a
class (C a) <= D a where
  d : a -> a
f : (C a) => a -> a
f = d
