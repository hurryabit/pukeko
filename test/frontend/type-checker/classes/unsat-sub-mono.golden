test/frontend/type-checker/classes/unsat-sub-mono.pu:6:1: cannot solve constraint D a in context (C a)
